### PR TITLE
change unsuccessful ep email copy for user

### PR DESCRIPTION
**Pull request by macool**
https://trello.com/c/eKrgQ02Y/631-qae-support-change-unsuccessful-ep-email-copy-for-as-they-do-not-get-feedback

### CODE REVIEW

**Comment by macool** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;eKrgQ02Y&#x2F;631-qae-support-change-unsuccessful-ep-email-copy-for-as-they-do-not-get-feedback"> QAE Support: Change UNSUCCESSFUL EP email copy for as they do not get feedback</a>

File: app/views/users/unsuccessful_feedback_mailer/ep_notify.text.slim [hunk 1]
```diff
@@ -1,15 +1,19 @@
 = "Dear #{@user.full_name},"
 
-' Firstly, we'd like to thank you for taking the time to nominate
+' THE QUEEN’S AWARD FOR ENTERPRISE PROMOTION 2016
```
Comment (matthewford) [inline app/views/users/unsuccessful_feedback_mailer/ep_notify.text.slim]: Um, isn't this supposed to be the title?

**Comment by matthewford** [comment]
Also both HTML and text versions need updating

File: app/views/users/unsuccessful_feedback_mailer/ep_notify.text.slim (hunk 1 above)
Comment (macool) [inline app/views/users/unsuccessful_feedback_mailer/ep_notify.text.slim]: @matthewford this is inside the email in the document. should I wipe this title?

**Comment by macool** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;3zHl7FAt&#x2F;633-sxt-support-if-pn-is-enabled-a-the-clinic-when-a-user-views-the-clinic-go-directly-to-the-pn-tab-not-the-clinic-details-tab"> SXT Support: If PN is enabled a the clinic - when a user views the clinic go directly to the PN tab not the clinic details tab.</a>

File: app/mailers/users/unsuccessful_feedback_mailer.rb [hunk 2]
```diff
@@ -13,7 +13,7 @@ def ep_notify(form_answer_id)
     @user = @form_answer.user.decorate
     @nominee_name = @form_answer.nominee_full_name
 
-    subject = "Queen's Award for Enterprise Promotion: Nomination Feedback"
+    subject = "Queen's Award for Enterprise: Promotion 2016"
```
Comment (matthewford) [inline app/mailers/users/unsuccessful_feedback_mailer.rb]: THE QUEEN’S AWARD FOR ENTERPRISE PROMOTION 2016

**Comment by macool** [comment]
@matthewford I've amended

**Pull request merged**